main.cpp

```diff
@@ -113,7 +113,9 @@ int main(void) {
   std::pair<float, float> info_coordinates(info_x, info_y);
 
   ftpServer *ftpServerInstance = nullptr;
+
   while (running) {
+
```

Comment:
I'm not sure this line break was intended?

Reply:
Nope, removed.